### ISSUE TITLE
大佬，提示Failed to canonicalize script path, Training has ended.

### ISSUE THREAD

**Issue by olubai** [licyk/sd-webui-all-in-one#6]
安装全部没问题，使用全部设置好后，点击开始训练按钮闪了一下就没反应了。
启动正常：
[2025-09-09 11:06:58][SD-Trainer Installer]:: 初始化中
[2025-09-09 11:06:58][SD-Trainer Installer]:: SD-Trainer Installer 版本: v3.0.8
[2025-09-09 11:06:58][SD-Trainer Installer]:: 当前内核路径前缀: core
[2025-09-09 11:06:58][SD-Trainer Installer]:: 完整内核路径: D:\kohya\core
[2025-09-09 11:06:58][SD-Trainer Installer]:: 使用默认 HuggingFace 镜像源
[2025-09-09 11:06:58][SD-Trainer Installer]:: 默认启用 uv 作为 Python 包管理器, 加快 Python 软件包的安装速度
[2025-09-09 11:06:58][SD-Trainer Installer]:: 当 uv 安装 Python 软件包失败时, 将自动切换成 Pip 重试 Python 软件包的安装
[2025-09-09 11:06:58][SD-Trainer Installer]:: 检测 uv 是否需要更新
[2025-09-09 11:06:58][SD-Trainer Installer]:: uv 无需更新
[2025-09-09 11:06:58][SD-Trainer Installer]:: 使用 PyPI 镜像源
[2025-09-09 11:06:58][SD-Trainer Installer]:: 检测到本地存在 launch_args.txt 启动参数配置文件 / -LaunchArg 命令行参数, 已读取该启动参数配置文件并应用启动参数
[2025-09-09 11:06:58][SD-Trainer Installer]:: 使用的启动参数: --inbrowser --language en-GB --noverify
[2025-09-09 11:06:58][SD-Trainer Installer]:: 检查 SD-Trainer 运行环境中
[2025-09-09 11:06:58][SD-Trainer Installer]:: 检查 SD-Trainer 内核依赖完整性中
[2025-09-09 11:06:58][SD-Trainer Installer]:: SD-Trainer 无缺失依赖
[2025-09-09 11:06:58][SD-Trainer Installer]:: 检测 PyTorch 的 libomp 问题中
[2025-09-09 11:06:58][SD-Trainer Installer]:: PyTorch 检查完成
[2025-09-09 11:06:58][SD-Trainer Installer]:: 检查 onnxruntime-gpu 版本问题中
[2025-09-09 11:06:59][SD-Trainer Installer]:: onnxruntime-gpu 无版本问题
[2025-09-09 11:06:59][SD-Trainer Installer]:: 检查 Numpy 版本中
[2025-09-09 11:06:59][SD-Trainer Installer]:: Numpy 无版本问题
[2025-09-09 11:06:59][SD-Trainer Installer]:: 检测 Microsoft Visual C++ Redistributable 是否缺失
[2025-09-09 11:06:59][SD-Trainer Installer]:: Microsoft Visual C++ Redistributable 未缺失
[2025-09-09 11:06:59][SD-Trainer Installer]:: SD-Trainer 运行环境检查完成
[2025-09-09 11:06:59][SD-Trainer Installer]:: 检测是否可设置 CUDA 内存分配器
[2025-09-09 11:06:59][SD-Trainer Installer]:: 设置 CUDA 内存分配器为 CUDA 内置异步分配器
[2025-09-09 11:06:59][SD-Trainer Installer]:: 启动 SD-Trainer 中
11:07:03-555833 WARNING  Skipping requirements verification.
11:07:03-557833 INFO     headless: False
11:07:03-557833 INFO     Using shell=True when running external commands...
* Running on local URL:  http://127.0.0.1:7860
* To create a public link, set `share=True` in `launch()`.

运行后台提示：
10:40:04-461108 INFO     Start training LoRA Standard ...
10:40:04-462285 INFO     Validating lr scheduler arguments...
10:40:04-463353 INFO     Validating optimizer arguments...
10:40:04-464308 INFO     Validating D:/kohya gui/core/outputs existence and writability... SUCCESS
10:40:04-465325 INFO     Validating
                         D:/lora-scripts-v1.12.0_Portable/lora-scripts/sd-models/illustriousXL_v01.safetensors
                         existence... SUCCESS
10:40:04-466289 INFO     Validating D:/kohya gui/core/dataset/images/oh existence... SUCCESS
10:40:04-466289 INFO     Folder 6_ohIL: 6 repeats found
10:40:04-467288 INFO     Folder 6_ohIL: 50 images found
10:40:04-467288 INFO     Folder 6_ohIL: 50 * 6 = 300 steps
10:40:04-468288 INFO     Regularization factor: 1
10:40:04-468288 INFO     Train batch size: 1
10:40:04-469288 INFO     Gradient accumulation steps: 1
10:40:04-469288 INFO     Epoch: 1
10:40:04-469288 INFO     Max train steps: 1600
10:40:04-470288 INFO     stop_text_encoder_training = 0
10:40:04-470288 INFO     lr_warmup_steps = 0.1
10:40:04-471288 INFO     Effective Learning Rate Configuration (based on GUI settings):
10:40:04-471288 INFO       - Main LR (for optimizer & fallback): 1.00e-04
10:40:04-472288 INFO       - Text Encoder (Primary/CLIP) Effective LR: 1.00e-04 (Fallback to Main LR)
10:40:04-472288 INFO       - Text Encoder (T5XXL, if applicable) Effective LR: 1.00e-04 (Fallback to Main LR)
10:40:04-473290 INFO       - U-Net Effective LR: 1.00e-04 (Specific Value)
10:40:04-473290 INFO     Note: These LRs reflect the GUI's direct settings. Advanced options in sd-scripts (e.g., block
                         LRs, LoRA+) can further modify rates for specific layers.
10:40:04-475288 INFO     Saving training config to D:/kohya gui/core/outputs\OH-IL0909k_20250909-104004.json...
10:40:04-475288 INFO     Executing command: D:\kohya gui/python/Scripts\accelerate.EXE launch --dynamo_backend no
                         --dynamo_mode default --mixed_precision fp16 --num_processes 1 --num_machines 1
                         --num_cpu_threads_per_process 2 D:/kohya gui/core/sd-scripts/sdxl_train_network.py
                         --config_file D:/kohya gui/core/outputs/config_lora-20250909-104004.toml
Failed to canonicalize script path
10:40:04-616352 INFO     Training has ended.
